builtin/logical/nomad/backend_test.go

```diff
@@ -27,7 +28,13 @@ func (c *Config) APIConfig() *nomadapi.Config {
 	return apiConfig
 }
 
-func prepareTestContainer(t *testing.T) (func(), *Config) {
+func (c *Config) Client() (*nomadapi.Client, error) {
+	apiConfig := c.APIConfig()
+
+	return nomadapi.NewClient(apiConfig)
+}
+
+func prepareTestContainer(t *testing.T, manuallyBootstrap bool) (func(), *Config) {
```

Comment:
nit: It is not immediately clear to me what "manually" means in this context. I assume it means the opposite of "automatic".

Would it be better to invert it and make it `autoACL` or something similar, perhaps?

Reply:
In light of https://github.com/hashicorp/vault/pull/10751 -- maybe naming it `bootstrap` would be preferable.